app/controllers/intake_forms_controller.rb

```diff
@@ -0,0 +1,37 @@
+class IntakeFormsController < ApplicationController
+  def new
+  end
+
+  def create
+    sponsor_group = SponsorGroup.create(sponsor_group_params)
+
+    if sponsor_group.persisted?
+      redirect_to root_path
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def sponsor_group_params
+    @sponsor_group_params ||= params.require(:intake_form).permit(
```

Comment:
No need to memoize the params as they are only used once, only memoize when required.